doc/source/drivers/vector/miramon.rst

```diff
@@ -0,0 +1,374 @@
+a set of point files and a set of arc files will be created.
+
+Consequently, during creation the MiraMon vector driver output can be a
+folder or a set of files with the appropriate extension (*.pnt*, etc):
+
+- If the output is a **folder**, it will contain all the translated layers with the original name in the origin dataset.
+
+  - In this case a *.mmm* file will be created referencing all layers in the origin dataset to make an
+    easy open of the dataset using the MiraMon software.
+  - In this case, please specify the MiraMon file output format name using the -f option (**-f MiraMonVector**).
+
+- If it the output is a **file** with extension all the translated layers in the origin dataset will be created with the specified name.
+  Use this option only when you know that there is only one layer with one feature type in the origin dataset.
+
+The attributes of the MiraMon feature are stored in an associated *.dbf*.
+If a classical DBF IV table could not be used (too many fields or records,
+large text fields, etc) a file type called extended DBF is used.
+This is an improvement of dBASE IV DBF files. The specification of this format can be found in `this file
+<https://www.miramon.cat/new_note/eng/notes/DBF_estesa.pdf>`__.
+
+Note that extended *.dbf* files cannot be opened with Excel or
+other typical programs. If the complete MiraMon Professional software
+is not installed on the computer, the free and standalone
+MiraD application can be downloaded from
+`this page <https://www.miramon.cat/USA/Prod-MiraD.htm>`__ to open them.
+
+Connection string
+-----------------
+
+The MiraMon driver accepts three types of sources of data:
+
+When translating from a MiraMon vector format, the MiraMon vector driver input needs a file with one of the
+described extensions: 
+
+-  *.pnt* for *points*.
+-  *.arc* for *stringlines*.
+-  *.pol* for *polygons* (or *multipolygons*).
+
+The extension *.nod* is not valid for translation. Take in consideration all auxiliary files described above.
+
+Field sizes
+-----------
+
+The driver knows to auto-extend string and integer fields to
+dynamically accommodate for the length of the data to be inserted.
+
+Size Issues
+-----------
+
+Geometry: The MiraMon vector format explicitly uses 32-bit offsets in the 1.1 version
+and 64-bit offsets in the 2.0 version. It is better to produce 1.1 version files if 2.0
+version is not really necessary than always use 2.0 version. Version 1.x files are smaller.
+
+Attributes: The dbf format does not have any offsets in it, so it can be
+arbitrarily large.
+
+Open options
+------------
+
+The following open options are available.
+
+-  .. oo:: Height
+      :choices: First, Lowest, Highest
+
+      Sets which of the possible heights for each vertex is read: 
+      the *first*, the *lowest* or the *highest* one. It only applies to
+      MiraMon multi-height layers, where the same X,Y vertex can have more than one Z.
+
+-  .. oo:: iMultiRecord
```

Comment:
Is the "i" prefix here intentional, or is it the internal API bleeding out?

Reply:
Hi,
It is intentional. I chose that name to indicate 'the index of.' But it can be changed to make it more in line with GDAL's nomenclature. This means that in a string/real/integer list field value of a feature, you might only want one of the elements of the list, perhaps because the output driver doesn't support lists.

Reply:
I've pushed a commit to rename the option to MultiRecordIndex

Reply:
Fine!